core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-favorite-page/dot-favorite-page.component.html

```diff
@@ -7,15 +7,34 @@
 >
```

Comment:
The save button of the dialog allows you to click N times, and the user doesn't know that it is saving; please use the loading state of the PrimeNg Button when you are in the background saving.

<img width="866" alt="Screenshot 2023-01-13 at 10 33 13 AM" src="https://user-images.githubusercontent.com/1909643/212358026-9853e576-e38d-41e4-950b-2cf0759491cf.png">

Reply:
fixed!

https://user-images.githubusercontent.com/37185433/212427683-7cf09340-0cdc-4626-8012-652190f8d55b.mov

Reply:
Perfecto.